include/seqan3/io/structure_file/output.hpp

```diff
@@ -492,7 +492,7 @@ class structure_file_output
      */
     template <std::ranges::input_range rng_t>
     structure_file_output & operator=(rng_t && range)
-        requires tuple_like<reference_t<rng_t>>
+        requires tuple_like<std::ranges::range_reference_t<rng_t>>
```

Comment:
So this is not your fault as it was an issue before. Please wrap the requires in `//!\cond` and `//!\endcond`, then the Doxygen warning should go away. Same for sequence_file/output